test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestQueryParserTest.cs

```diff
@@ -196,6 +196,7 @@ public static void PopulateNodeForExpression(Expression expression, XmlNode node
         [TestMethod]
         public void RequestQueryParserIdentifiersAmbiguous()
         {
+            Assert.Fail();

```

Comment:
what's this for?

Reply:
I'm ensuring that the tests are being run in the pipeline build line correctly. This is just a draft for testing, though